src/React InstantSearch/public/index.html

```diff
@@ -1,18 +1,22 @@
 <!doctype html>
 <html lang="en">
-  <head>
-    <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-    <!-- Do not use @4 in production, use a complete version like x.x.x, see website for latest version:
-      https://community.algolia.com/react-instantsearch/Getting_started.html
+
+<head>
```

Comment:
Just wondering, why the indent change?

Reply:
It’s my Prettier setup for the HTML files. I will revert the changes.